incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -28,21 +28,41 @@ COPY ./user-app/pom.xml /project/user-app/pom.xml
 RUN cd /project/user-app && ./validate.sh
 
 # Copy the rest of the application source
-COPY ./user-app/src /project/user-app/src
+COPY ./user-app /project/user-app
```

Comment:
Since we're copying everything, which might include a /target (even a /target/wlp) from running in the host.... we should probably change line 36 below to do a clean:
`mvn -B liberty:create package`

Reply:
added a clean step

Reply:
OK, that was my bad idea... since it eliminates the benefit of having done the `install-server` above.

Even if there is no overwriting, there is still the possibility you get different results depending on what you happened to do on the host or not which would be confusing, so we should do this.

What we should use is **.dockerignore** to exclude target.   Actually it already does :)  so just remove the clean then.
----

Also, I know this is a bigger change so am open to doing it later, but...   

We don't need the whole 150MB         <liberty.artifactId>openliberty-runtime</liberty.artifactId>   for this... we just need the kernel.   That's be a nice way to slim down a bit.. we can use another profile.